content/altinn-studio/app-creation/configuration/eformidling/_index.md

```diff
@@ -0,0 +1,186 @@
+
+I tillegg må det i samme fil opprettes en ny seksjon; _EFormidlingClientSettings_.
+Innholdet i kodesnutten nedenfor kan kopieres i sin helhet. 
+Denne setter opp url til integrasjonspunktet.
+Lenken peker på mocken som kan kjøres opp lokalt.
+Les mer om oppsettet av eFormidlings mocken [her](#-Kjøring-med-eFormidling-mock-lokalt).
+
+Når en app deployes til test eller produksjon vil denne verdien overskrives
+og peke mot integrasjonspunktet i Altinn Platform.
+
+
+```json
+"EFormidlingClientSettings": {
+   "BaseUrl": "http://localhost:9093/api/"
+ }
+```
+
+Dersom det ikke er ønskelig å teste integrasjonen med eFormidling når man kjører applikasjonen lokalt kan man overstyre
+denne konfigurasjonen i `appsettings.Development.json`. 
+Opprett _AppSettings_ seksjonen dersom den ikke finne og sett `EnableEFormidling` til false.
+
+```json
+"AppSettings": {
+    "EnableEFormidling": false
+}
+```
+
+## Konfigurere nøkkelverdier for eFormidling i applikasjonen din 
+
+Det kreves en del metadata om eFormidlingsforsendelsen og denne defineres i `applicationmetadata.json`.
+Filen finner du i repoet under mappen `App/config`.
+
+Opprett en ny seksjon `eFormidling` og fyll ut verdier for følgende parametre.
+
+| Id| Beskrivelse|
+|-----|-------------|
+| serviceId |  |
+| process | prosessid som settes på scopet i StandardBusinessDocumentHeader|
+| dataTypes | Liste id for data typer som skal legges ved forsendelsen. |
+| sendAfterTaskId | Id på tasken som skal avsluttes før forsendelsen sendes. Det er anbefalt at dette er et confirmation steg|
+| receiver | Organsisasjonsnummer til mottaker. Støtter kun norske virksomheter. Kan sløyfes og defineres i applogikken |
+| standard | DocumentIdentification standard |
+| type | DocumentIdentification type |
+| typeVersion | DocumentIdentification type versjon |
+| securityLevel | Sikkerhetsnivå som settes på StandardBusinessDocument |
+
+
+Et eksempel for en konfguasjon i application metadata:
+
+```json
+"eFormidling": {
+    "serviceId": "DPF",
+    "process": "urn:no:difi:profile:arkivmelding:administrasjon:ver1.0",
+    "dataTypes": [ "default" ],
+    "sendAfterTaskId": "Task_2",
+    "receiver": "910075918",
+    "standard": "urn:no:difi:arkivmelding:xsd::arkivmelding",
+    "type": "arkivmelding",
+    "typeVersion": "2.0",
+    "securityLevel":  3
+  }
+```
+
+## Generering av metadata til forsendelsen i applikasjonen
+
+I App.cs må man overstyre metoden som henter ut den standardrekkefølgen av sider som er definert i `Settings.json`
+Dette gjøres ved å legge til funksjonen nedenfor i App.cs.
+Forventet output fra denne metoden er en tuppel som inneholder navnet på metadatafilen som første element
+og en stream med metadataen som andre element.
+
+```cs
+/// <inheritdoc />
+public override async Task<(string, Stream)> GenerateEFormidlingMetadata(Instance instance)
+{
+    Altinn.Common.EFormidlingClient.Models.Arkivmelding arkivmelding = new Altinn.Common.EFormidlingClient.Models.Arkivmelding();  
```

Comment:
We can (and should) use `var` in this case.

Reply:
Also some more cases below.

Reply:
Updated. Not sure which other cases you are referring to that should be updated as well. Point me in the right direction, and I'll update these as well.